types/index.d.ts

```diff
@@ -305,7 +352,7 @@ export namespace OperationOptions {
     authorize?: boolean | TrustLineFlag;
   }
   interface ChangeTrust extends BaseOptions {
-    asset: Asset;
+    asset: ChangeTrustAsset;
```

Comment:
Need a similar change for `TrustLineAsset` and `RevokeTrustlineSponsorship`, right?

Reply:
Good catch! Added the missing type change in https://github.com/stellar/js-stellar-base/pull/454/commits/6bba221f910a322d1979ab1737c17296622aa731.